packages/cli/src/executable/AgenticaStart.ts

```diff
@@ -1,20 +1,27 @@
 import chalk from "chalk";
+import cp from "child_process";
 import fs from "fs/promises";
-import inquirer from "inquirer";
+import inquirer, { QuestionCollection } from "inquirer";
 import path from "path";
 import prettier from "prettier";
 import typia from "typia";
 
 import { Connector } from "../bases/Connector";
 import { Package } from "../bases/Package";
 import { Tsconfig } from "../bases/Tsconfig";
+import { IAgenticaStart } from "../structures/IAgenticaStart";
 import { createProjectDirectory } from "../utils/createProjectDirectory";
 import { getNpmPackages } from "../utils/getNpmPackages";
-import { getQuestions } from "../utils/getQuestions";
+import { PackageManager } from "../utils/types/PackageManager";
+import { ProjectOptionValue } from "../utils/types/ProjectOption";
 
 export namespace AgenticaStart {
-  export async function execute(input: { projectName: string }) {
-    const { projectName } = input;
+  /**
+   * Execute `start` command.
+   */
+  export async function execute(input: IAgenticaStart.IExecuteInput) {
```

Comment:
Nit, I recommend destructuring on parameter statement
```suggestion
  export async function execute({ projectName, options }: IAgenticaStart.IExecuteInput) {
```